clusterloader2/pkg/util/cluster.go

```diff
@@ -96,24 +96,20 @@ func isNodeUntainted(node *corev1.Node) bool {
 	return true
 }
 
-func isNodeConditionSetAsExpected(node *corev1.Node, conditionType corev1.NodeConditionType, wantTrue, silent bool) bool {
+func isNodeConditionSetAsExpected(node *corev1.Node, conditionType corev1.NodeConditionType, wantTrue bool) bool {
 	// Check the node readiness condition (logging all).
 	for _, cond := range node.Status.Conditions {
 		// Ensure that the condition type and the status matches as desired.
 		if cond.Type == conditionType {
 			if wantTrue == (cond.Status == corev1.ConditionTrue) {
 				return true
 			}
-			if !silent {
-				klog.Infof("Condition %s of node %s is %v instead of %t. Reason: %v, message: %v",
-					conditionType, node.Name, cond.Status == corev1.ConditionTrue, wantTrue, cond.Reason, cond.Message)
-			}
+			klog.V(4).Infof("Condition %s of node %s is %v instead of %t. Reason: %v, message: %v",
```

Comment:
There are couple more places in perf-tests repo that do that - can you please also fix them (or open an issue)?

Reply:
https://github.com/kubernetes/perf-tests/issues/980